Formula/p/pypy3.10.rb

```diff
@@ -72,6 +72,22 @@ def install
       s.gsub! "/include'", "/include/tcl-tk'"
     end
 
+    if OS.mac?
+      # Allow python modules to use ctypes.find_library to find homebrew's stuff
+      # even if homebrew is not a /usr/local/lib. Try this with:
+      # `brew install enchant && pip install pyenchant`
+      inreplace "lib-python/3/ctypes/macholib/dyld.py" do |f|
+        f.gsub! "DEFAULT_LIBRARY_FALLBACK = [",
+                "DEFAULT_LIBRARY_FALLBACK = [ '#{HOMEBREW_PREFIX}/lib', '#{Formula["openssl@3"].opt_lib}',"
+        f.gsub! "DEFAULT_FRAMEWORK_FALLBACK = [", "DEFAULT_FRAMEWORK_FALLBACK = [ '#{HOMEBREW_PREFIX}/Frameworks',"
+      end
+
+      # Fix mismatch in test definition of ncurses
+      inreplace "lib_pypy/_curses_build.py" do |f|
+        f.gsub! "char * tparm(const char *, ...);", "char * tparm(char *, ...);"
+      end
```

Comment:
Have you reported this upstream?

Reply:
I will do - I suspect it's specific to mac / the option which controls whether tparm is a varargs fn.  I need to figure out the right reporting mechanism for pypy

Reply:
[resolved a](https://github.com/pypy/pypy/issues/4967)